src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionxpathfilter/InputSuppressionXpathFilterEscapeChar.java

```diff
@@ -0,0 +1,18 @@
+package com.puppycrawl.tools.checkstyle.filters.suppressionxpathfilter;
+
+/*
+ * Config: Default.
+ */
+public class InputSuppressionXpathFilterEscapeChar { // violation
```

Comment:
I doubt there is a violation :)

Reply:
Yeah, sorry. I was trying all possibilities at that time. Done now.